src/components/login/CasLogin.js

```diff
@@ -30,4 +32,12 @@ module.exports = {
         window.location.href = casUrl;
     },
 
-};
+    render: function() {
+        return (
+            <div>
+                <button onClick={this.onCasClicked}>Sign in with CAS</button>
+            </div>
+        );
+    }
+
+});
```

Comment:
So this is the new non-split controller structure? I guess we've also ditched the atomic design categories? I don't have a problem with this per-se (organising them by function rather than by level is probably sensible) but as long as we're aware this is happening.